specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchservice.json

```diff
@@ -5784,6 +5784,82 @@
     },
+    "Semantic": {
+      "properties": {
+        "semanticConfigurations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SemanticConfiguration"
+          },
+          "description": "The semantic configurations for the index."
+        }
+      },
+      "externalDocs": {
+        "url": "https://docs.microsoft.com/en-us/azure/search/semantic-search-overview"
+      },
+      "description": "Defines parameters for a search index that influence semantic capabilities."
+    },
+    "SemanticConfiguration": {
+      "properties": {
+        "name": {
+          "externalDocs": {
+            "url": "https://docs.microsoft.com/rest/api/searchservice/Naming-rules"
+          },
+          "type": "string",
+          "description": "The name of the semantic configuration.",
+          "x-nullable": false
+        },
+        "prioritizedFields": {
+          "$ref": "#/definitions/PrioritizedFields",
+          "x-nullable": false,
+          "description": "Describe how the service should process document fields in the context of semantic capabilities.",
+        }
+      },
+      "required": [
+        "name",
+        "prioritizedFields"
+      ],
+      "externalDocs": {
+        "url": "https://docs.microsoft.com/en-us/azure/search/semantic-search-overview"
+      },
+      "description": "Defines a specific configuration to be used in the context of semantic capabilities."
+    },
+    "PrioritizedFields": {
+      "properties": {
+        "TitleField": {
+          "$ref": "#/definitions/SemanticField",
+          "description": "Define which field to be used as the title of the document in the context of semantic capabilities."
+        },
+        "prioritizedContentFields": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SemanticField/"
+          },
+          "description": "Define the fields to be processed in the context of semantic capabilities. For the best result, the selected fields should contain text in natural language form. The order of the fields in the array represent their priority. Fields with lower priority may get truncated if the content is too long."
+        },
+        "PrioritizedKeywordsFields": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SemanticField/"
+          },
+          "description": "Define the fields to be processed in the context of semantic capabilities. For the best result, the selected fields should contain text in keyword form. The order of the fields in the array represent their priority. Fields with lower priority may get truncated if the content is too long."
+        }
+      },
+      "externalDocs": {
+        "url": "https://docs.microsoft.com/en-us/azure/search/semantic-search-overview"
+      },
+      "description": "Describe how the service should process document fields in the context of semantic capabilities."
```

Comment:
"Specify whether the service performs semantic processing on content fields or keyword fields."  (I might not be reading the JSON correctly, but in general, specific is better.  If descriptions are too general, then the reader will have low confidence in your doc and will have to keep looking around.

Reply:
@HeidiSteen - one quick clarification. Customers can set all three. Bing models expect content, title, and keywords/url. That's why we've broken them out into three separate lists. It's not one or the other

Reply:
How about: "Defines the keyword fields to be used for semantic ranking, captions, highlights, and answers. For the best result, the selected fields should contain a list of keywords. The order of the fields in the array represent their priority. Fields with lower priority may get truncated if the content is long."

Reply:
How about "Describes the title, content, and keyword fields to be used for semantic ranking, captions, highlights, and answers."

Reply:
I like the description, but I'm wondering about re-using the same one for multiple APIs. Shouldn't each description be distinct?